README.md

```diff
@@ -19,7 +31,7 @@ TypeScript declarations are provided within the project.
 - Uses out-of-the-box default options to ease use and keep code short
 - Provides hot reloading for schema and options
 - Provides out-of-the-box metrics for GraphQLServer
-- Uses and is compatible with `graphql-js` library version 14 and 15.
+- Uses and is compatible with `graphql-js` library version 15 and 16.
```

Comment:
Is this version still compatible with `graphql-js` 15?

Reply:
The application v 1.x is compatible with graphql-js 15 while v 2.x is compatible with graphql-js 16.

Reply:
OK, would it make sense to add that here, even though we have the table above this text?

Reply:
Added the info